modules/server_new/src/main/scala/observe/server/tcs/TcsSouth.scala

```diff
@@ -223,48 +224,30 @@ object TcsSouth {
     guideConfigDb:       GuideConfigDb[F]
   )(
     targets:             TargetEnvironment,
-    stepConfig:          StepConfig,
+    telescopeConfig:     CoreTelescopeConfig,
     lightPath:           LightPath,
     observingWavelength: Option[Wavelength]
   ): TcsSouth[F] = {
+    val p: Offset.P = telescopeConfig.offset.p
+    val q: Offset.Q = telescopeConfig.offset.q
 
-    val gwp1   = none.flatMap(_ =>
-      StepConfig.science.andThen(StepConfig.Science.guiding).getOption(stepConfig)
-    )
-    val gwp2   = none.flatMap(_ =>
-      StepConfig.science.andThen(StepConfig.Science.guiding).getOption(stepConfig)
-    )
-    val gwoi   = none.flatMap(_ =>
-      StepConfig.science.andThen(StepConfig.Science.guiding).getOption(stepConfig)
-    )
-    val gwc1   = none.flatMap(_ =>
-      StepConfig.science.andThen(StepConfig.Science.guiding).getOption(stepConfig)
-    )
-    val gwc2   = none.flatMap(_ =>
-      StepConfig.science.andThen(StepConfig.Science.guiding).getOption(stepConfig)
-    )
-    val gwc3   = none.flatMap(_ =>
-      StepConfig.science.andThen(StepConfig.Science.guiding).getOption(stepConfig)
-    )
-    val gwod1  = none.flatMap(_ =>
-      StepConfig.science.andThen(StepConfig.Science.guiding).getOption(stepConfig)
-    )
-    val gwod2  = none.flatMap(_ =>
-      StepConfig.science.andThen(StepConfig.Science.guiding).getOption(stepConfig)
-    )
-    val gwod3  = none.flatMap(_ =>
-      StepConfig.science.andThen(StepConfig.Science.guiding).getOption(stepConfig)
-    )
-    val gwod4  = none.flatMap(_ =>
-      StepConfig.science.andThen(StepConfig.Science.guiding).getOption(stepConfig)
-    )
+    val guiding: StepGuideState = telescopeConfig.guiding
+
+    val gwp1   = none.map(_ => guiding)
```

Comment:
Wouldn't the `map` never be called?

Reply:
Exactly. I don't know why code was this way, I supposed it is a reminder for when we work on the other instruments. I tried to respect that.

Reply:
@jluhrs why's this?